RELEASE.rst

```diff
@@ -106,6 +108,25 @@ Etherscan verification is documented here: https://github.com/raiden-network/rai
 Note that we currently have some issues with the script: https://github.com/raiden-network/raiden-contracts/issues/349.
 
 
+.. _measure-gas:
+
+Measure Gas Costs
+-----------------
+
+``raiden_contracts`` package provides `some constants <https://github.com/raiden-network/raiden-contracts/blob/de13cf9aa7ad7ed230ff204e47103def6a14b0be/raiden_contracts/constants.py#L35>`__ showing the amount of gas that each operation requires. This information is manually updated. The amounts can be measured with a script
+
+::
+
+    pytest -s raiden_contracts/tests/test_print_gas.py
+
+The script prints many numbers like
```

Comment:
@pirapira Wouldn't it make more sense to remake the script into `measure_gas.py` and have it auto-generate the `constants.py` for you instead of you manually creating by copying the output of the script?

Reply:
That's https://github.com/raiden-network/raiden-contracts/issues/16.